tests/test_pipeline/test_transforms_3d.py

```diff
@@ -34,6 +34,57 @@ def test_remove_points_in_boxes():
     assert points.shape == (10, 4)
 
 
+def test_object_sample():
+    import pickle
+    db_sampler = mmcv.ConfigDict({
+        'data_root': './tests/data/kitti/',
+        'info_path': './tests/data/kitti/kitti_dbinfos_train.pkl',
+        'rate': 1.0,
+        'prepare': {
+            'filter_by_difficulty': [-1],
+            'filter_by_min_points': {
+                'Pedestrian': 10
+            }
+        },
+        'classes': ['Pedestrian', 'Cyclist', 'Car'],
+        'sample_groups': {
+            'Pedestrian': 6
+        }
+    })
+    with open('./tests/data/kitti/kitti_dbinfos_train.pkl', 'rb') as f:
+        db_infos = pickle.load(f)
+    np.random.seed(0)
+    object_sample = ObjectSample(db_sampler)
+    points = np.fromfile(
+        './tests/data/kitti/training/velodyne_reduced/000000.bin',
+        np.float32).reshape(-1, 4)
+    annos = mmcv.load('./tests/data/kitti/kitti_infos_train.pkl')
+    info = annos[0]
+    annos = info['annos']
+    gt_names = annos['name']
+    gt_bboxes_3d = db_infos['Pedestrian'][0]['box3d_lidar']
+    gt_bboxes_3d = LiDARInstance3DBoxes([gt_bboxes_3d])
+    CLASSES = ('Car', 'Pedestrian', 'Cyclist')
+    gt_labels = []
+    for cat in gt_names:
+        if cat in CLASSES:
+            gt_labels.append(CLASSES.index(cat))
+        else:
+            gt_labels.append(-1)
+    input_dict = dict(
+        points=points, gt_bboxes_3d=gt_bboxes_3d, gt_labels_3d=gt_labels)
+    input_dict = object_sample(input_dict)
+    points = input_dict['points']
+    gt_bboxes_3d = input_dict['gt_bboxes_3d']
+    gt_labels_3d = input_dict['gt_labels_3d']
+    repr_str = repr(object_sample)
+    expected_repr_str = 'ObjectSample'
```

Comment:
We may refine the `__repr__` of the `ObjectSample` module to print more useful information.